stacks/textToSpeechUsingElevenLabs/index.test.ts

```diff
@@ -0,0 +1,8 @@
+import LetYourCuriosityGuideYou from ".";
+
+test("Convert this text to speech: Let your curiosity guide you", async () => {
+  const response = await LetYourCuriosityGuideYou();
```

Comment:
pass a parameter to this function and call the function, because otherwise it's pretty unlikely people would need this function. Stackwise should normally call this function something like "convertTTS"